src/tools/pkcs11-tool.c

```diff
@@ -4847,7 +4847,7 @@ static int encrypt_decrypt(CK_SESSION_HANDLE session,
 		return 0;
 
 	if (EVP_PKEY_size(pkey) > (int)sizeof(encrypted)) {
-		fprintf(stderr, "Ciphertext buffer too small\n");
+		printf("Ciphertext buffer too small\n");
```

Comment:
why print errors on stdout instead of stderr?

Reply:
It is not an error, but part of the "testsuite report", which starts on line 4843, does not end with a newline and expects some result to be added (which is in this case "error").

    printf("    %s: ", p11_mechanism_to_name(mech_type));

We can go the other way round and print everything on the stderr too. But if part is printed to the first stream and the second part is printed to the second stream, the formatting is undefined.